_data/meghanabhange/meghana.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Meghana Bhange
+institution: MKSSS's Cummins College of Engineering
+profile_pic: meghana.png 
+quote: "2020 is giving me a fiction-esque sense of humor"
```

Comment:
Remove double quotes from begin and end.

Reply:
Removed the double quotes :) d2a4c35